dev/tools/localization/bin/gen_localizations.dart

```diff
@@ -65,8 +66,10 @@ String generateArbBasedLocalizationSubclasses({
   required String baseClass,
   required HeaderGenerator generateHeader,
   required ConstructorGenerator generateConstructor,
+  ConstructorGenerator? generateConstructorForCountrySubClass,
   required String factoryName,
   required String factoryDeclaration,
+  required bool factoryIsConst,
```

Comment:
wondering: why are they not all const?

Reply:
this bool decide whether to add `const` keyword when calling factory. Since material/cupertino pass in parameter to when calling factory, they can't use the `const`. Will update the name to be more precise